sdk/management/pom.xml

```diff
@@ -91,6 +91,12 @@
                 <artifactId>azure-storage-blob</artifactId>
                 <version>12.4.0</version>
             </dependency>
+            <dependency>
+                <groupId>com.github.spotbugs</groupId>
+                <artifactId>spotbugs-annotations</artifactId>
+                <version>4.0.0-beta3</version>
```

Comment:
Any reason we pick this beta version? The latest is [4.0.2](https://mvnrepository.com/artifact/com.github.spotbugs/spotbugs-annotations/4.0.2).

Reply:
This is aligned with the plugin version. https://github.com/azure/azure-sdk-for-java/blob/5810f30086c93a1431deeae7f0a10f890b0012d6/sdk/management/pom.xml#L221, I'm not sure if these should be aligned or not. Maybe we could discuss about.

Reply:
I've change it to latest version